### PR TITLE
deployment and helm charts update for release 1.5.3

### PR DESCRIPTION
**Pull request by aramase**
**Reason for Change**:

- Update deployment manifests
- Update helm chart for release 1.5.3
- Update changelog for 1.5.3

**Issue Fixed**:

**Notes for Reviewers**:

### CODE REVIEW

File: deploy/infra/deployment-rbac.yaml [hunk 1]
```diff
@@ -64,16 +64,16 @@ rules:
   verbs: ["get", "list"]
 - apiGroups: [""]
   resources: ["pods"]
-  verbs: ["get", "list"]
+  verbs: ["get", "list", "watch"]
```
Comment (kkmsft) [inline deploy/infra/deployment-rbac.yaml]: MiC role updates are missing ?

File: charts/aad-pod-identity/templates/nmi-clusterrole.yaml [hunk 2]
```diff
@@ -15,16 +15,16 @@ rules:
   verbs: ["get", "list"]
 - apiGroups: [""]
   resources: ["pods"]
-  verbs: ["get", "list"]
+  verbs: ["get", "list", "watch"]
```
Comment (kkmsft) [inline charts/aad-pod-identity/templates/nmi-clusterrole.yaml]: MIC role updates ?

File: deploy/infra/deployment-rbac.yaml (hunk 1 above)
Comment (aramase) [reply deploy/infra/deployment-rbac.yaml]: There are no role updates for mic.

File: charts/aad-pod-identity/templates/nmi-clusterrole.yaml (hunk 2 above)
Comment (aramase) [reply charts/aad-pod-identity/templates/nmi-clusterrole.yaml]: No role updates for MIC. MIC already had the required roles.

**Comment by evenh** [comment]
It would be nice to include #416 in 1.5.3

**Pull request merged**